src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -12,8 +13,11 @@
 #include "openvic-simulation/types/Date.hpp"
 #include "openvic-simulation/types/IdentifierRegistry.hpp"
 #include "openvic-simulation/types/IndexedMap.hpp"
+#include "openvic-simulation/types/fixed_point/FixedPoint.hpp"
+#include "openvic-simulation/types/Sliders.hpp"
 #include "openvic-simulation/utility/Getters.hpp"
 
+
```

Comment:
Could you remove the extra newline here? In general we only ever use at most a single empty line between blocks of code